### ISSUE TITLE
Install and configure

### ISSUE THREAD

**Issue by etnpoolorg** [Snipa22/nodejs-pool#367]
I will help you to install quickly and efficiently.
The price is 0.005 BTC
My telegram @mrmircrypto

**Comment by sircryptojohn** [comment]
Take caution on this.

MiR ran minepool.live and exit scammed a bunch of AEON miners after he hit ~250kh/s or more.

Additionally, many experienced pool operators will offer assistance at no charge.

**Comment by bobbieltd** [comment]
@sircryptojohn Are you out of mind ? No experienced pool operators offer you assitance at no charge. LOL. Give me any names with TOP POOLs.

**Comment by bobbieltd** [comment]
@etnpoolorg Are you nuts or idiots ? No one will pay you. Here is not marketplace for you to sell your rubbish pools.

**Comment by BKdilse** [comment]
I agree with @sircryptojohn , most experienced guys I've dealt with have been helpful, with no cost associated.  I'm no expert, but try and help where I can :)

**Comment by LegolasGChief** [comment]
@bobbieltd  @arqtras helped me

**Comment by BKdilse** [comment]
@Venthos @Bathmat @ArqTras @snipa22 mainly and others.

**Comment by Venthos** [comment]
In the grand scheme of things, 0.005 BTC (currently $42 USD) isn't a bad deal.

Even Snipa22's Readme file shows he'd help for 7 XMR (installation assistance) or 4 XMR (config assistance).  At the time he wrote those values (Aug 14, 2017) XMR was about $47.  So Snipa22 was offering installation for roughly $329 USD and configuration for roughly $188 USD respectively.

However, I imagine the trust in the original developer's guidance (especially if you intend to provide them with access to anything) has much more value than someone unknown that is advertising their services on what should otherwise be a bug report/feature request system.

You can generally find folks (like myself) willing to offer tips and guidance, as long as you are willing and capable of "doing the work" and it's not too demanding.  If you are looking for someone to configure everything or do a full install and configure, I imagine most would be looking for some sort of compensation for their time.

**Comment by BKdilse** [comment]
I agree totally.

**Comment by ArqTras** [comment]
If somebody can’t install poo using even simple deploy bash ... make no sense. Better read here, digg and try to learn something. And if shit happen don’t You worry about opinion to be a stealer ? Pool owner skills are very important. Where is sense runing pool without dev team ?

**Comment by Chinsusu** [comment]
someone pls help me? I stuck at login into admin.html. I've done everything as the guide

**Comment by ArqTras** [comment]
Read manual on first page. There is everything described

**Comment by ArqTras** [comment]
Read manual on first page. There is everything described, specially info about login to admin.html

**Comment by Chinsusu** [comment]
I've tried almost 10 times. but still stuck there. Do I need to connect daemon to run the cp admin?

**Comment by LearnMiner** [comment]
> @mrmircrypto

have discord?

**Comment by LearnMiner** [comment]
> I will help you to install quickly and efficiently.
> The price is 0.005 BTC
> My telegram @mrmircrypto

have discord?
